src/nrc/api/serializers.py

```diff
@@ -183,8 +184,10 @@ def _send_to_subs(self, msg: dict):
             if group.match_pattern(msg_filters):
                 subs.add(group.abonnement)
 
-        task_kwargs = {}
-        if settings.LOG_NOTIFICATIONS_IN_DB:
+        task_kwargs = {"notificatie_id": notificatie_id}
```

Comment:
doesn't this result in the celery task now _always_ saving the notification?

Reply:
Hmm, it would do that yes, but only if you manually send a Notification (by creating one via the admin)